CMSgov/design-system
# [WNMGDS-2674] Add VRTs for basic examples

## Summary

- Add VRTs for basic examples (not the astro one)
  - Right now we have an unresolved issue with building the astro project where it can't find a couple of the dependencies and always asks you to install them. The build is successful, but it fails. [This issue was supposed to have been fixed](https://github.com/withastro/astro/issues/8230) in the previous version of Astro, but it still seems to be a problem for us. It could be because of our monorepo structure and yarn.
- Reorganize snapshot files into subfolders so they're easier to navigate

## How to test

`yarn test:browser:examples`

In order to see the few files that actually changed, you'll need to filter out pngs in the diff. GitHub doesn't seem to respect the filter url args when it first loads, so there's still a good chance to crashing the page even if you [follow this filtered link](https://github.com/CMSgov/design-system/pull/2959/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.ts&show-deleted-files=true&show-viewed-files=true).

## Checklist
